RevenueCatUI/Templates/V2/Components/Stack/StackComponentViewModel.swift

```diff
@@ -128,6 +151,7 @@ struct StackComponentStyle {
         self.shape = shape?.shape
         self.border = border?.border
         self.shadow = shadow?.shadow
+        self.badge = badge?.badge(stackShape: self.shape)
```

Comment:
The stack shape is only used to calculate the the corner radius of the edge-to-edge leading/trailing style badge, and make it match the parent stack.